services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_logs.py

```diff
@@ -85,7 +85,7 @@ async def stop_log_fetching(self, container_name: str) -> None:
             return
 
         task.cancel()
-        with suppress(CancelledError):
+        with suppress(CancelledError, TimeoutError):
```

Comment:
I would add your justification here "Sometimes the docker engine will raise TimeoutError when awaiting the log fetching task in the background"